### PR TITLE
Update api posts with link type

### PR DESCRIPTION
**Pull request by squigglybob**
add the link field type to the post api list of fields

### CODE REVIEW

**Comment by squigglybob** [comment]
@corsacca this is the docs for the api links

**Comment by squigglybob** [comment]
We had said not to worry about force values in the link type PR but I can
put it in if you'd like.

I can put a set example, I had rolled update and set together here by
saying that the meta id is only needed in that first one if you are
updating. It's probably clearer to split them apart though

**Comment by corsacca** [comment]
@squigglybob We can just remove the "force_values" from this documentation piece

File: theme-core/api-posts/post-types-fields-format.md [hunk 1]
```diff
@@ -60,6 +60,24 @@ $fields = [
...
+* none as yet
+
+```php
+$fields = [
+  "social_links" => [
+    "values" => [ 
+      [  "type" => "fb", "value" => "facebook.com", "meta_id" => "1234" ],  //set a value, the type must be predefined in the field options, meta_id is only needed if updating a value
+      [ "meta_id" => "1234", "delete" => true ] //remove existing
+    ],
+    "force_values" => false // true will set source to the values entries. removing all others
```
[the remark this answers is not in the document]
Comment (squigglybob) [reply theme-core/api-posts/post-types-fields-format.md]: i've removed force values for now

File: theme-core/api-posts/post-types-fields-format.md [hunk 2]
```diff
@@ -60,6 +60,24 @@ $fields = [
...
+
+* none as yet
+
+```php
+$fields = [
+  "social_links" => [
+    "values" => [ 
+      [  "type" => "fb", "value" => "facebook.com", "meta_id" => "1234" ],  //set a value, the type must be predefined in the field options, meta_id is only needed if updating a value
+      [ "meta_id" => "1234", "delete" => true ] //remove existing
+    ],
```
[the remark this answers is not in the document]
Comment (squigglybob) [reply theme-core/api-posts/post-types-fields-format.md:76]: i've split the top one into 2

**Comment by corsacca** [comment]
Thanks @squigglybob !

**Pull request merged**